app/code/Magento/Bundle/Test/Unit/Pricing/Price/ConfiguredPriceTest.php

```diff
@@ -0,0 +1,185 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Bundle\Test\Unit\Pricing\Price;
+
+use Magento\Bundle\Pricing\Price\DiscountCalculator;
+use Magento\Catalog\Model\Product;
+use Magento\Catalog\Model\Product\Configuration\Item\ItemInterface;
+use Magento\Bundle\Pricing\Price\ConfiguredPrice;
+use Magento\Bundle\Pricing\Adjustment\Calculator;
+use Magento\Catalog\Pricing\Price\ConfiguredPriceSelection;
+use Magento\Framework\DataObject;
+use Magento\Framework\Pricing\Amount\AmountInterface;
+use Magento\Framework\Pricing\Price\PriceInterface;
+use Magento\Framework\Pricing\PriceCurrencyInterface;
+use Magento\Framework\Pricing\PriceInfo\Base;
+use Magento\Framework\Serialize\Serializer\Json;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Test for \Magento\Bundle\Pricing\Price\ConfiguredPrice
+ */
+class ConfiguredPriceTest extends TestCase
+{
+    /**
+     * @var float
+     */
+    private $basePriceValue = 100.;
```

Comment:
Is there any reason of not specifying the decimals as well (and in the other parts as well)?

Reply:
> Hi @monteshot, please check the below suggestions. Looks like it were missed some of the previous ones.
> Moreover, could you please provide the exact steps to reproduce the mentioned issue (maybe some piece of code)? Just to be easier to test this one.
> 
> Thank you.

This code executes on bundle product page. But i think the code in the configured price class is some type of fallback.

This code invokes in renderrer in \Magento\Catalog\Pricing\Render\FinalPriceBox block
![image](https://user-images.githubusercontent.com/2836146/125645063-3fbb489e-ec8c-4efa-a5f0-3ef5011ea0d1.png)

I did debug only if i emulate some data that will allow proceed to the if statement
![image](https://user-images.githubusercontent.com/2836146/125646776-e7837656-4e87-42bf-a67a-c6d8fd5f7720.png)
![image](https://user-images.githubusercontent.com/2836146/125646851-43fc21e8-28d1-44cf-ad96-4eb69defea26.png)

@eduard13

Reply:
![image](https://user-images.githubusercontent.com/2836146/125647074-27ff67de-93bf-456a-a66a-5a7a9fc1d508.png)

Reply:
> Is there any reason of not specifying the decimals as well (and in the other parts as well)?

No reason. I will add decimal after the point

Reply:
Additionally you may check the original task to get steps to reproduce
https://github.com/magento/magento2/issues/33334#issue-929044829